e2e/artefacts/k8s/vsp-ds.yaml

```diff
@@ -29,7 +29,7 @@ spec:
         securityContext:
           privileged: true
         command: [ "/usr/bin/ipuplugin" ]
-        args: [ "--bridgeType=linux", "--interface=enp0s1f0d3", "--ovsCliDir=/opt/p4/p4-cp-nws", "-v=debug"]
+        args: [ "--bridgeType=linux", "--interface=enp0s1f0d3", "--portMuxVsi=0x0b", "--ovsCliDir=/opt/p4/p4-cp-nws", "-v=debug"]
```

Comment:
We should replace this with the second octet from the apf associated with enp0s1f0d3

Reply:
This is the 2nd octet on this interface on both IPUs in our OCP setup. I this can be different on other systems but this parameters here can be overridden if that's the case.
```
IPU-1:
-------
enp0s1f0d3       UP             00:0b:00:03:03:18 <BROADCAST,MULTICAST,UP,LOWER_UP> 

IPU-2:
-------
enp0s1f0d3       UP             00:0b:00:03:03:3c <BROADCAST,MULTICAST,UP,LOWER_UP> 
```

Reply:
I think that should be fine, we can create a separate PR for this to remove the portmux option. The reason I think it needs to be removed is that it is redundant since we use a dedicated apf for internal portmux enp0s1f0d3